src/cli/cli_tcp.cpp

```diff
@@ -273,6 +307,24 @@ template <> otError TcpExample::Process<Cmd("deinit")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli tcp bind
+ * @code
+ * tcp bind :: 30000
+ * Done
+ * @endcode
+ * @cparam tcp bind @ca{ip} @ca{port}
+ * * `ip`: IPv6 address to bind to. If you wish to have the TCP/IPv6 stack assign
+ *    the binding IPv6 address, use the following value to use the
+ *    unspecified IPv6 address: `::`.
+ * * port: TCP port number to bind to.
+ * @par
+ * Associates an IPv6 address and a port to the example TCP endpoint provided by
+ * the `tcp` CLI. Associating the TCP endpoint to an IPv6
+ * address and port is referred to as naming the TCP endpoint. This binds the
```

Comment:
"naming the TCP endpoint."  (should be in quotes)

Not sure if we really need this here, unless we're referencing this elsewhere (i.e. using "naming the TCP endpoint" in other commands or the upcoming guide)

Reply:
going to use it in Concepts too...made similar statement in UDP.  I added the quotes.